### PR TITLE
Formalized Dual Hitl Hootl Minutia (t/0311)

### PR DESCRIPTION
**Pull request by shihaocao**
# Formalized Dual Hitl Hootl Minutia 

This PR cherrypicks the changes out of Dual Hitl Hootl that are core to a streamlined integrated test

### Summary of changes (Shihao will make this list exhaustive ASAP)
- PTest-PSim sided autotelem is now enabled iff 'autotelem' exists somewhere in the device config file.
- != string comparisons are fixed
- DualPsim is added
- State field report uses IMEI number now instead of "FlightControllerFollower"
- QuakeAutotelem is added
- Add orbit.vel to telemetry
- Make PropFaultHandler compatible with constructor changes

### Testing
Trying to get some of the changes of dual hitl hootl in

### CODE REVIEW

File: ptest/cases/dual_empty_case.py [hunk 1]
```diff
@@ -20,7 +20,8 @@ def fast_boot_follower(self):
         return True
 
     def run_case_fullmission(self):
-        self.finish()
+        pass
+        #self.finish()
```
Comment (shihaocao) [inline ptest/cases/dual_empty_case.py:24]:
```suggestion
        self.finish()
```

File: ptest/radio_session.py [hunk 2]
```diff
@@ -223,7 +223,7 @@ def send_uplink(self):
 
         if success:
             # Send the uplink to Iridium
-            to = "data@sbd.iridium.com"
+            to = "pan.ssds.qlocate@gmail.com"
```
Comment (shihaocao) [inline ptest/radio_session.py:226]:
```suggestion
            to = "data@sbd.iridium.com"
```

File: ptest/psim/simulation.py [hunk 3]
```diff
@@ -46,9 +46,18 @@ def __init__(self, is_interactive, devices, seed, testcase, sim_duration,
         self.scrape_emails = scrape_emails
         self.log = ""
 
+        if device_config != None and 'autotelem' in str(device_config):
```
Comment (shihaocao) [inline ptest/psim/simulation.py:49]:
```suggestion
        # if the json config has devices, and the string 'autotelem' is somewhere in the dictionary
        if device_config != None and 'autotelem' in str(device_config):
```

File: ptest/usb_session.py [hunk 4]
```diff
@@ -441,7 +441,7 @@ def dbtelem(self):
             field: value,
                 "time": str(datetime.datetime.now().isoformat())
             })
-            res = self.es.index(index='statefield_report_'+str(self.device_name.lower()), doc_type='report', body=data)
+            res = self.es.index(index='statefield_report_'+str(self.radio_imei), doc_type='report', body=data)
```
Comment (shihaocao) [inline ptest/usb_session.py:444]: Using IMEI numbers now

**Review by Duncan-McD — APPROVED** [review]

**Pull request merged**